### PR TITLE
Cairo 2.9.2

### PR DESCRIPTION
**Pull request by edg-l**
## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Review by azteca1998 — DISMISSED** [review]

File: tests/common.rs [hunk 1]
```diff
@@ -256,7 +256,7 @@ pub fn run_native_program(
 pub fn run_vm_program(
     program: &(String, Program, SierraCasmRunner),
     entry_point: &str,
-    args: &[Arg],
+    args: Vec<Arg>,
```
[the remark this answers is not in the document]
Comment (edg-l) [reply tests/common.rs:259]: The problem is Arg is not clone, so Into doesnt work, which is why i changed it to Vec

[the remark this answers is not in the document]
Comment (azteca1998) [reply tests/common.rs:259]: I see, that's annoying. Nothing we can do about that then.

**Review by azteca1998 — APPROVED** [review]

**Review by azteca1998 — DISMISSED** [review]

**Comment by FrancoGiachetta** [comment]
The starknet-replay's version inside the blocks-running workflow should be changed to make it work properly. This is its most recent one: cccbe7514cb9e83e429850e0516d8865888ba947

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Environment detection.
 
 UNAME := $(shell uname)
-CAIRO_2_VERSION = 2.9.0-dev.0
-SCARB_VERSION = 2.8.4
+CAIRO_2_VERSION = 2.9.2
+SCARB_VERSION = 2.9.2 # todo update once scarb 2.9.2 is released
```
Comment (azteca1998) [inline Makefile:5]:
```suggestion
SCARB_VERSION = 2.9.2
```

**Review by azteca1998 — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -66,7 +66,7 @@ test: check-llvm needs-cairo2 build-alexandria runtime-ci
 	cargo test --profile ci --features=scarb,with-cheatcode,with-debug-utils
 
 .PHONY: test-cairo
-test-cairo: check-llvm needs-cairo2 build-alexandria runtime-ci
```
Comment (FrancoGiachetta) [inline Makefile:69]: Why removing the alexandría build? Does their PR updating cairo-lang work?

**Review by azteca1998 — APPROVED** [review]

File: Makefile (hunk 3 above)
Comment (edg-l) [reply Makefile:69]: i didnt remove it, just from test-cairo, which tests the corelib, so its not needed for this target

Comment (FrancoGiachetta) [reply Makefile:69]: Oh okay

**Review by FrancoGiachetta — APPROVED** [review]

**Pull request merged**